src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -508,6 +522,34 @@ bool WholeBodyDynamicsDevice::openExternalWrenchesPorts(os::Searchable& config)
     return ok;
 }
 
+bool WholeBodyDynamicsDevice::openFilteredFTPorts(os::Searchable& config)
+{
+    bool ok = true;
+        // create port names
+
+        std::string sensorName;
+        std::string portName;
+        outputFTPorts.resize(estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE));
+        for(int ft=0; ft < estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE); ft++ )
+        {
+            sensorName= estimator.sensors().getSensor(iDynTree::SIX_AXIS_FORCE_TORQUE,ft)->getName() ;
+
+            yDebug() << "wholeBodyDynamics: creating port names for  " << sensorName;
+
+            portName=(portPrefix+"/filteredFT/"+sensorName);
+            //std::unique_ptr<yarp::os::BufferedPort<yarp::sig::Vector>> outputPort
```

Comment:
If this  is not used, we can remove it.

Reply:
Done in a83c188a98e2bd4f75f58a484361ed2a88fffea7